l10n_it_causali_pagamento/README.rst

```diff
@@ -0,0 +1,82 @@
+========================================
```

Comment:
Potresti mettere anche la spiegazione inglese come in https://github.com/OCA/l10n-italy/tree/10.0/l10n_it_vat_registries ?